src/UI/words.tsx

```diff
@@ -433,6 +439,13 @@ const dict = {
   "environment.protected.tooltip":
     "This action is not allowed because the environment is protected",
 
+  /**
+   * Provides views
```

Comment:
Provides views? These are related to the composer, so I'd keep them under that label.

Reply:
removed